### ISSUE TITLE
fund_bond_holdings not returning expected data

### ISSUE THREAD

**Issue by Eldomir** [dpguthrie/yahooquery#223]
It's possible this is because of a change in Yahoo's API, or I might just be daft. But for every bond ETF I know, the duration and maturity returned seem wrong to me. For an obvious example, MINT (an ultra short term bond ETF) shows this:

{
    'creditQuality': {},
    'creditQualityCat': {},
    'duration': 4.79,
    'durationCat': {},
    'maturity': 98.72,
    'maturityCat': {}
}

**Comment by dpguthrie** [comment]
Do you know where this data is contained on YF?  I'm having trouble finding where it's located to confirm (I imagine that you're right, the duration seems pretty long for an ultra short term bond ETF).

**Comment by Eldomir** [comment]
No, on the YF website I can't see duration or maturity for bond ETFs (I might just not be looking in the right place) . I'd expect to find it on the profile, like https://finance.yahoo.com/quote/MINT/profile?p=MINT, but I don't see it there.
